src/components/shared/Pill.js

```diff
@@ -0,0 +1,34 @@
+import { Box, Paragraph } from 'grommet'
+
+// status: error, info, warning
+
+export const Pill = ({
+  background = '',
+  color = '',
+  label = '',
+  labelOnly = false,
+  status = ''
+}) => {
+  return (
+    <Box
+      align="center"
+      background={status ? `${status}-bg` : background}
+      direction="row"
+      flex
+      pad={{ horizontal: 'xsmall', vertical: 'xxsmall' }}
+      round="30px"
+      width="max-content"
+    >
+      {!labelOnly && (
```

Comment:
I would change the name of this to be `dot` and when true, have it show the dot.

Reply:
Completed, thank you!